react/Table/Virtualized/Dnd/TableRow.jsx

```diff
@@ -17,8 +17,9 @@ const TableRow = ({ item, context, ...props }) => {
     canDrag: canDragProps,
     dragId
   } = dragProps
+  const { isCut } = props
   const isSelected = context?.isSelectedItem(item)
-  const isDisabled = itemsInDropProcess.includes(item._id)
+  const isDisabled = itemsInDropProcess.includes(item._id) || isCut
```

Comment:
```suggestion
  const isDisabled = itemsInDropProcess.includes(item._id)
```

Let's not mix drag and drop with our newly added prop.
It should be better to just use it like this for example

```tsx
<TableRowClassic
  {...props}
...
  disabled={isDisabled || isDimmed}
  hover
/>
```